org-code-javabuilder/media/src/main/java/org/code/media/SoundException.java

```diff
@@ -1,13 +1,14 @@
 package org.code.media;
 
+import org.code.media.support.SoundExceptionKeys;
 import org.code.protocol.JavabuilderRuntimeException;
 
 public class SoundException extends JavabuilderRuntimeException {
-  protected SoundException(SoundExceptionKeys key) {
+  public SoundException(SoundExceptionKeys key) {
```

Comment:
This doesn't go in support?

Reply:
`SoundException` is thrown by `SoundLoader` which is currently in the documentation for students.